src/modules/Tab/Tab.js

```diff
@@ -37,6 +39,9 @@ class Tab extends Component {
     /** Shorthand props for the Menu. */
     menu: PropTypes.object,
 
+    /** Shorthand props for the Grid. */
+    grid: PropTypes.object,
```

Comment:
Shorthand can be a string, number, props object, or an element.  It is then passed to a shorthand factory, such as `Grid.create(grid)`.

In this case, the user cannot pass normal Grid props such as `celled` or `padded` because this object is not used in a shorthand factory for the Grid. This is some kind of configuration rather than shorthand.

I'd like to reserve the `grid` prop for actual Grid props so users can control that if needed.  That said, I'm not so sure of the best API for these values.  Here are some ideas, would love feedback:

### Top level props
Make `paneWidth` and `tabWidth` top level props rather than nested in `grid`.  This makes room for `grid` to be true shorthand, however, those props are only applicable if `menu.vertical` is set so it might be a bit odd as well.

### Omit and use rest for shorthand
The current `grid` object needs to be spread on the `Grid` in order to be true Grid props.  We could leave it as is and just omit `paneWidth` and `tabWidth` from the `grid` value before spreading it:

```jsx
const { paneWidth, tabWidth, ...gridProps } = grid

<Grid {...gridProps>
  {GridColumn.create({ width: tabWidth })}
  {GridColumn.create({ width: paneWidth })}
</Grid>
```